lib/active_record/connection_adapters/postgis/oid/spatial.rb

```diff
@@ -11,9 +11,12 @@ class Spatial < Type::Value
           #   "geography(Point,4326)"
           #   "geometry(Polygon,4326) NOT NULL"
           #   "geometry(Geography,4326)"
-          def initialize(oid, sql_type)
-            @sql_type = sql_type
-            @geo_type, @srid, @has_z, @has_m = self.class.parse_sql_type(sql_type)
+          def initialize(geo_type: 'geometry', srid: 0, has_z: false, has_m: false, geographic: false)
```

Comment:
The documentation should be updated as well

Reply:
Yes, good point I'll do that!